lib/dateFormat.js

```diff
@@ -0,0 +1,12 @@
+export function dateFormat (timeValue) {
+
+    let date = new Date(timeValue)
+    date = date.toJSON()
```

Comment:
Function refactor: use Regex with replace:

```js
let date = new Date(timeValue).toJSON();
formatedDate = date.replace(/[-:\.]/g, "")
return formatedDate;
```

O `[ ]` na regex significa qualquer um dos caracteres e a `\` antes do ponto é pq o ponto é um identifier de qualquer caracter, se quisermos que ele detecte o ponto basta colocar a \ antes. Assim fica mais limpo e menos custoso.

Reply:
If you want to check it: https://regex101.com/r/o0bfVV/1